core/jobs.py

```diff
@@ -630,10 +630,11 @@ def run(self, job_id, job_class_str, output):
                     results_list.append(json.loads(item))
             job_class.register_completion(job_id, results_list)
         except Exception as e:
-            logging.error(traceback.format_exc())
```

Comment:
Why are we removing the traceback? is that expected?

Reply:
Traceback is logged through the `exception` now.